include/boost/beast/http/fields.hpp

```diff
@@ -245,7 +258,8 @@ class basic_fields
         The state of the moved-from object is
         as if constructed using the same allocator.
     */
-    basic_fields& operator=(basic_fields&&);
+    basic_fields& operator=(basic_fields&&) noexcept(
+        alloc_traits::propagate_on_container_move_assignment::value);
```

Comment:
Should also require `is_nothrow_move_assignable<Allocator>::value` (in addition to POCMA).

Reply:
This is part of the requirements of the Allocator concept. Should we check them with static_asserts?

Reply:
For the latter, if in C++17 or higher mode, it should be in the noexcept condition. i.e. `noexcept(alloc_traits::propagate_on_container_move_assignment::value || alloc_traits::is_always_equal::value)`.

Before C++17 the Allocator requirements were a little more relaxed. You could use `noexcept(alloc_traits::propagate_on_container_move_assignment::value && is_nothrow_move_assignable<Allocator>::value)` if in C++11/14 mode, but in practice I think you're fine with what you have (and just `static_assert`).